src/js/pages/catalog/PackageDetailTab.js

```diff
@@ -257,12 +322,12 @@ class PackageDetailTab extends mixin(StoreMixin) {
     }
 
     const cosmosPackage = CosmosPackagesStore.getPackageDetails();
-    if (state.isLoading || !cosmosPackage) {
+    const cosmosPackageVersions = CosmosPackagesStore.getPackageVersions();
+    if (state.isLoading || !cosmosPackage || !cosmosPackageVersions) {
```

Comment:
I wonder if we could avoid blocking the entire page and progressively load this, especially since the `cosmosPackageVersions` are only required/shown once you click the dropdown.

Reply:
It's addressed @MatApple